tests/utils.ts

```diff
@@ -5,150 +5,242 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import { ServerlessMysql } from "serverless-mysql";
-import { APIGatewayProxyEvent, Context } from "aws-lambda";
+import { ServerlessMysql } from 'serverless-mysql';
+import { APIGatewayProxyEvent, Context } from 'aws-lambda';
 import { IValidatedAPIGatewayProxyEvent } from '../src/libs/api-gateway';
+import { v4 } from 'uuid';
 
 export const cleanDatabase = async (mysql: ServerlessMysql): Promise<void> => {
-    const TABLES = [
-        'proposals',
-    ];
-    await mysql.query('SET FOREIGN_KEY_CHECKS = 0');
-    for (const table of TABLES) {
-        await mysql.query(`DELETE FROM ${table}`);
-    }
-    await mysql.query('SET FOREIGN_KEY_CHECKS = 1');
+  const TABLES = [
+    'proposals',
+    'websockets',
+  ];
+  await mysql.query('SET FOREIGN_KEY_CHECKS = 0');
+  for (const table of TABLES) {
+    await mysql.query(`DELETE FROM ${table}`);
+  }
+  await mysql.query('SET FOREIGN_KEY_CHECKS = 1');
 };
 
-
 /**
  * Creates a Handler Context object, for use on tests invoking lambdas
  */
 export function generateHandlerContext(): Context {
```

Comment:
Maybe we should move these utils that generate big objects to a `fixtures.ts` so this file can be more concise.
What do you think?

Reply:
It's a good idea. Created it on [e8daf02](https://github.com/HathorNetwork/hathor-atomic-swap-service/pull/5/commits/e8daf02be875a3d171e321a644aa5031c3569ae7).